src/build.py

```diff
@@ -784,8 +780,9 @@ def BuildEnv(build_dir, use_gnuwin32=False, bin_subdir=False,
 
 def LLVM():
   buildbot.Step('LLVM')
-  Mkdir(LLVM_OUT_DIR)
-  cc_env = BuildEnv(LLVM_OUT_DIR, bin_subdir=True)
+  out_dir = os.path.join(work_dirs.GetBuild(), 'llvm-out')
```

Comment:
How about:

```build_dir = os.path.join(work_dirs.GetBuild(), 'llvm')```

Reply:
Currently the default for `GetBuild` is just `src/work` and `src/work/llvm` is the LLVM source directory. So that would be a break from the current configuration. We can talk about doing that but maybe not in this PR.

Reply:
Oh i see.  It certainly makes sense to me that if you going distinguish between these different directories that they should not overlap.